gammapy/background/fov.py

```diff
@@ -3,3 +3,60 @@
 Field-of-view (FOV) background estimation
 """
 from __future__ import print_function, division
+from astropy.wcs import WCS
+from astropy.wcs.utils import pixel_to_skycoord
+import numpy as np
+from scipy.interpolate import interp1d
+
+__all__ = ['fill_acceptance_image',
+           ]
+
+
+def fill_acceptance_image(image, center, offset, acceptance):
+    """Generate a 2D image of a radial acceptance curve.
+
+    The radial acceptance curve is given as an array of values
+    defined at the specified offsets.
+
+    Parameters
+    ----------
+    image : `~astropy.io.fits.ImageHDU`
+        Empty image to fill.
+    center : `~astropy.coordinates.SkyCoord`
+        Coordinate of the center of the image.
+    offset : `~numpy.ndarray` of `~astropy.coordinates.Angle`
+        Array of offset values in degrees where acceptance is defined.
```

Comment:
No need to require "in degrees".
Users should be allowed to pass in `Angle` in any units they like (usually `radian` or `deg`).
If you really need some units internally, e.g. `deg`, you can convert to those units.